pages/3d-scatter-plots.py

```diff
@@ -4,16 +4,17 @@
 
 
 dash.register_page(
-    __name__, description="Iris samples filtered by petal width with a slider"
+    __name__, description="Iris samples in a scatter_3d plot filtered by petal width with a RangeSlider. This app has a top-bottom layout and a regular-callback."
 )
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
+#### Dash Components in the App:
+- [RangeSlider](https://plotly.com/python/range-slider/)
```

Comment:
This rangeslider is a dash core component. Can you please update the link to the rangeslider component in the dash docs. https://dash.plotly.com/dash-core-components/rangeslider